woocommerce/woocommerce
# [Accessibility] Add address title to edit/add buttons on My Account page

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #43631 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. On the frontend, go to My Account -> Addresses page
2. Turn on the screen reader.
3. Focus on the edit billing address button.
![image](https://github.com/woocommerce/woocommerce/assets/9420947/cc168020-2c7c-47fa-9833-a998fdd696bb)
4. Verify the screen reader announces "Edit Billing address".
5. Focus on the add shipping address button.
![image](https://github.com/woocommerce/woocommerce/assets/9420947/4ce3861c-d3a0-4d99-ab2d-b3ee9f5c14ff)
6. Verify the screen reader announces "Add Shipping address".



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
